packages/renderer/src/components/BasicComponents/FloatingModal.vue

```diff
@@ -2,7 +2,7 @@
   <!-- Opaque Overlay behind Modal -->
   <div
     v-if="props.showModal"
-    class="absolute inset-0 z-40 opacity-25 bg-black"
+    class="absolute inset-0 z-40 opacity-25 bg-black "
```

Comment:
We should really try to clean up such changes as the superflous space at the end here. This does not need to be changed.